.github/workflows/connectors.yml

```diff
@@ -27,4 +27,4 @@ jobs:
 
       - name: Check all connectors are installable
         run: |
-          ./install.sh --acceptlicenses --connectors=all
+          make pipelinewise all_connectors -e pw_acceptlicenses=y
```

Comment:
Isn't it enough to install only connectors, not the pipelinewise?

Reply:
if it's possible, then yeah sure